src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/GlobalTextareaChoiceListEntry.java

```diff
@@ -259,6 +259,13 @@ public boolean equals(Object o){
         return true;
     }
     
+    @Override
+    public int hashCode() {
+        int r = (name != null)?name.hashCode():0;
+        r = r * 31 + ((choiceList != null)?choiceList.hashCode():0);
+        return r;
+    }
+    

```

Comment:
HE_EQUALS_USE_HASHCODE
jp.ikedam.jenkins.plugins.extensible_choice_parameter.GlobalTextareaChoiceListEntry defines equals and uses Object.hashCode()